src/assetindex/assetIndex.es6

```diff
@@ -38,163 +31,152 @@ export const init = (li) => {
     });
 }
 
-const processMarketSubmarkets = (markets) => {
-    markets = menu.extractFilteredMarkets(markets);
+const state = {
+    dropdown: {
+        display_markets: null,
+        display_submarkets: null,
+        is_volatility: false,
+        market_submarkets: null,
+        selected_market: null
+    },
+    table: {
+        asset_data: [],
+        asset_data_extract: [],
+        display_headers: [],
+        display_asset_data: [],
+        search_input: ''
+    },
+    search: function (event, model) {
+        model.table.display_asset_data = [];
```

Comment:
model is the same as state here? so perhaps we could call it state.table to make it more readable + const declaration at the top + add space

Reply:
im not sure which space is it? is it in the array bracket?